src/components/Tokens/TokenDetails/ChartSection.tsx

```diff
@@ -60,7 +60,7 @@ function Chart({
   return (
     <ChartContainer data-testid="chart-container">
       <ParentSize>
-        {({ width }) => <PriceChart prices={prices ?? null} width={width} height={436} timePeriod={timePeriod} />}
+        {({ width }) => <PriceChart prices={prices} width={width} height={392} timePeriod={timePeriod} />}
```

Comment:
previously, PriceChart was arbitrarily adding subtracting 44px from the passed height. this oddity was leftover from previous refactor so I removed it.